modules/getting-started/pages/build-a-hello-world-application.adoc

```diff
@@ -81,41 +83,37 @@ include::{examplesdir}/build-a-hello-world-application.xml[]
 +
 . Click the *Save* icon to save your changes.
 
-You have now built a simple Mule application.
+You have now created a simple Mule application.
 
 == Deploy and Test Your Project Locally
 
-Anypoint Studio has an embedded Mule runtime that you use to test deployments. You can deploy your application on the embedded server to see it working and perform any debugging activities before deploying to a production environment, such as an on-prem server or Runtime Manager.
+Anypoint Studio has an embedded Mule runtime that you can use to test deployments. You can deploy your application on the embedded server to see it working and perform any debugging activities before deploying to a production environment, such as an on-premises server or Runtime Manager.
 
-. In the Package Explorer, right-click the project name, then select Run As > Mule Application.
+. In the Package Explorer, right-click the project name, then select *Run As > Mule Application*.
 +
-Studio launches the application on the embedded server, displaying its actions in the console. When complete, the console message reads:
-+
-"Started app 'basic_tutorial'"
+Studio launches the application on the embedded server, displaying its actions in the Console. When complete, the console message displays:
 +
 image::console1.png[console output]
 +
 . Open your favorite Web browser.
 . Navigate to the URL of your application.
 +
-This address is the result of the default settings of the HTTP connector:
+The URL is the result of the default settings of the HTTP connector:
 +
```

Comment:
result sounds weird. What about "The URL is created using the values of (the related?) default settings of the HTTP connector:" That's kind of awkward. But something more concrete than "is the result"?

But also, default settings? Customer can't control these? i.e. if they change the settings, does the URL change?